http-custom-mechanism/pom.xml

```diff
@@ -150,8 +154,64 @@
             <plugin>
                 <groupId>org.wildfly.plugins</groupId>
                 <artifactId>wildfly-maven-plugin</artifactId>
+                <version>${version.plugin.wildfly}</version>
             </plugin>
         </plugins>
     </build>
-
+    <profiles>
+        <profile>
+            <id>provisioned-server</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.wildfly.plugins</groupId>
+                        <artifactId>wildfly-maven-plugin</artifactId>
+                        <configuration>
+                            <feature-packs>
+                                <feature-pack>                                    <location>org.wildfly:wildfly-galleon-pack:${version.server}</location>
+                                </feature-pack>
+                            </feature-packs>
+                            <layers>
+                                <!-- layers may be used to customize the server to provision -->
+                                <layer>cloud-server</layer>
+                            </layers>
+                            <!-- deploys the quickstart on root web context -->
```

Comment:
you need the configure-elytron.cli when provisioning

Reply:
Added!